docs/advanced/api_examples.yml

```diff
@@ -281,6 +281,29 @@
 
 -
   id:
+    "create container with environment"
+
+  doc: |
+    Create a new container.
+
+  request: |
+    POST /v1/configuration/containers HTTP/1.1
+
+    {"host": "%(NODE_0)s",
+     "name": "webserver",
+     "image": "nginx:latest",
+     "environment": {"CONFIG_FILE": "/etc/nginx/nginx.conf", "USER": "webserver"}}
+
+  response: |
+    HTTP/1.1 201 Created
+
+    {"host": "%(NODE_0)s",
+     "name": "webserver",
+     "image": "nginx:latest",
+     "environment": {"CONFIG_FILE": "/etc/nginx/nginx.conf", "USER": "webserver"}}    
```

Comment:
This pretty printed response isn't strictly what the server will return. 
I see that the other container API examples are the same.
But in the future, when the we test these examples automaticallyu, will they pass or fail.
It also makes me think that the example was hand crafted rather than actually tested.